test/services/invoice_rebilling_create_transaction.service.test.js

```diff
@@ -132,4 +132,14 @@ describe('Invoice Rebilling Create Transaction service', () => {
       })
     })
   })
+
+  async function addRebillInvoice (billRunId, customerReference, financialYear, rebilledInvoiceId, rebilledType) {
```

Comment:
How would you feel about putting this actually on the `InvoiceHelper` instead? It's a personal preference but if I can within tests I like to try and segregate logic/code into support methods and keep the test files focused on tests. 🤷 🤔